custom-routes-vis/src/main/webapp/script.js

```diff
@@ -28,10 +28,34 @@ const MarkerNames = {
  * @enum {string}
  */
 const url_ids = {
-  IOS_URL_ID: 'ios-url', 
+  IOS_DEV_URL_ID: 'ios-dev-url', 
+  IOS_PRODUCTION_URL_ID: 'ios-production-url',
   V1_ANDROID_URL_ID: 'v1-android-url', 
   V2_ANDROID_URL_ID: 'v2-android-url',
```

Comment:
We usually keep the common part of the name in the front.
IOS_URL_ID_DEV
IOS_URL_ID_PRODUCTION
ANDROID_URL_ID_V1
ANDROID_URL_ID_V2